senders/victorops/send.go

```diff
@@ -0,0 +1,112 @@
+package victorops
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/writeas/go-strip-markdown"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/senders/victorops/api"
+)
+
+// SendEvents implements Sender interface Send
+func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.ContactData, trigger moira.TriggerData, plot []byte, throttled bool) error {
+	var messageType api.MessageType
+	switch events[len(events)-1].State {
+	case moira.StateERROR:
+		messageType = api.Critical
+	case moira.StateEXCEPTION:
+		messageType = api.Critical
+	case moira.StateWARN:
+		messageType = api.Warning
+	case moira.StateOK:
+		messageType = api.Recovery
+	case moira.StateNODATA:
+		messageType = api.Info
+
+	}
+
+	createAlertRequest := api.CreateAlertRequest{
+		MessageType:       messageType,
+		StateMessage:      sender.buildMessage(events, trigger, throttled),
+		EntityDisplayName: sender.buildTitle(events, trigger),
+		StateStartTime:    events[len(events)-1].Timestamp,
+	}
+	err := sender.client.CreateAlert(contact.Value, createAlertRequest)
+	if err != nil {
+		return fmt.Errorf("error while sending alert to victorops: %s", err)
+	}
+	return nil
+}
+
+func (sender *Sender) buildMessage(events moira.NotificationEvents, trigger moira.TriggerData, throttled bool) string {
+	var message strings.Builder
+	desc := stripmd.Strip(trigger.Desc)
+	eventsString := sender.buildEventsString(events, -1, throttled)
+	message.WriteString(desc)
+	message.WriteString(eventsString)
+	return message.String()
+}
+
+func (sender *Sender) buildTitle(events moira.NotificationEvents, trigger moira.TriggerData) string {
+	title := string(events.GetSubjectState())
+
+	tags := trigger.GetTags()
+	if tags != "" {
+		title += " " + tags
+	}
+
+	triggerURI := trigger.GetTriggerURI(sender.frontURI)
+	if triggerURI != "" {
+		title += fmt.Sprintf(" %s|%s", triggerURI, trigger.Name)
```

Comment:
Are you sure that link formatting works here? You can throw off the documentation, I did not find it =(

Reply:
If it does not work, then we can insert a link to a trigger in a free field (for example, to a trigger_url)

Reply:
we can do the same with reference to the picture, create a custom field, for example image_url